### DISCUSSION TITLE
Too beginner

### DISCUSSION THREAD

**Discussion by im-ellipsism** [community/community#46539] [New to GitHub] [answered]
### 1. A little about me.

_(You can skip this paragraph, not describe the problem. For that, you can go directly to point 2.)_

> Hi everyone, I had the audacity to want to use GitHub, but I am feeling very frustrated! I am old in the world of programming, but all my life I have programmed in Cobol because in Chile it is widely used in the financial system and there are few professionals who program in that language.
> 
> I haven't had the opportunity to learn programming with today's languages, but I thought I could work on GitHub, where the smartest programmers in the world live.

### **2. The problem**

If possible, I need help to create a website with Wordpress. But that is not the main problem. I don't know how to configure anything, neither domains nor DNS's nor external hostings nor GitHub's. By this I mean free domains and hosting, since I know how to handle payments well, but I don't have the budget to do it.

I hope I am not wrong by asking for such basic help for someone who knows nothing..

Note: I've read all the reference documentation on what I want to do, but it wasn't entirely clear to me yet.

I really appreciate your support.

Sandra.

By the way, my English is not perfect so I use Google translator :S

**Comment by Juanescacha** [comment, marked as answer]
Hola creo que hablas español! jeje dejame te explico un poquito
no se mucho de wordpress pero se que lo puedes montar en cualquier pagina que de hosting grauito para proyectos de aprendizaje:
- Vercel.com
- Netlify.com
- Render.com

y con ellas lo puedes conseguir facilmente para subir tu codigo y que este disponible super facil y rapido en la web

te recomiendo esta guia que explican como hacerlo en vercel con wordpress y mas tecnologias
https://vercel.com/guides/wordpress-with-vercel

cualquier duda que tengas me avisas! espero esto te ayude

**Reply by im-ellipsism** [reply]
jajaja y yo que me esforcé tanto escribiendo en inglés.

Muchas gracias Juan Esteban. He intentado probar varios hostings gratuitos pero no tengo un dominio "formal", y ninguno me aceptaba los .tk, pero esos que mencionas no los había visto. Aunque mi propósito era aprender a usar el hosting de Github, la información que me has dado me va a ser de mucha utilidad, así que ahora mismo me pongo a hacer la tarea.

Gracias!!!!

**Reply by Juanescacha** [reply]
Con mucho gusto! jajaja
recuerda que como estos sistemas son gratuitos te daran una url como por ejemplo esta: 
ya si quieres tu puedes conseguir pagando o gratis otro dominio y puedes hacer que ese hosting tenga el dominio personalizado, recuerda que el dominio no mas es el url y pero tambien debe haber un servidor 24/7 que este hostiando tu pagina a cualquier dominio en este caso como los que te recomende.

Y por ultimo me falto decirte el mas facil Github Pages: https://docs.github.com/es/pages es una forma rapida y sencilla de montar una pagina con tu cuenta de github, subes los archivos siguiendo las instrucciones de la guia a un repositorio y ya quedaria automaticamente desplegada la pagina

**Reply by im-ellipsism** [reply]
Te lo agradezco mucho, sobre todo porque fuiste la única persona que se tomó la molestia de contestar, y por supuesto por la información que me proporcionaste.

**Reply by Juanescacha** [reply]
Con mucho gusto! no hay de que ;)

**Reply by gitchacan** [reply]
Admire determination!  Stay focused and you will accomplish this..  Don't be easily discouraged.  Set a goal for each day even if its small.  Stay in the game. 
 Good luck to you.

**Reply by Hezekimostul** [reply]
- [ ]

**Comment by Sir-K-Grind** [comment]
I don’t know how to use codeQl but I have a cve in my possession that someone put profile

**Comment by ghost** [comment]
Ich weiß nicht ob ich hier auf der richtigen Seite bin.
Ich wollte die Steuerung meiner Wallbox mit EVCC unterstützen. Um zu testen, ob das funktioniert habe ich einen Token für 2 Euro erworben. Wenn ich auf der EVCC Seite versuche den Token abzuholen bekomme ich die Auskunft, dass kein Token gefunden wurde.
Hat da jemand eine Idee
